packages/firestore/test/util/api_helpers.ts

```diff
@@ -152,7 +154,8 @@ export function querySnapshot(
     mutatedKeys,
     fromCache,
     syncStateChanged,
-    false
+    false,
+    resumeToken
```

Comment:
Just specify `ByteString.EMPTY_BYTE_STRING` directly here, instead of assigning it to a local variable first. This is a bit more readable. Just use that `*/ resumeToken=*/` comment style to document what it is.